src/api/status/public/index.html

```diff
@@ -410,6 +409,27 @@ <h4 class="mb-0">$103,430</h4>
               </div>
             </div>
           </div>
+          <div class="col-xl-3 col-sm-6 mb-xl-0 mb-4 mt-4">
```

Comment:
Think this should wait for @menghif  PR to have the same style?

Reply:
For sure. I just copy paste the same component of the page for testing posts count.

Reply:
Can you match this card with all the previous cards? And if you don't have a footer, use `card-body` instead of `card-header`. Your current card is losing the rounded bottom corners.

Reply:
This needs to get addressed per @Andrewnt219's comment.

Reply:
@Andrewnt219 So i will need to wait for @menghif PR to have the same style?

Reply:
It was merged 2 days ago. The PR is already on your branch. Just need to copy the structure from the card with `card--telescope` on it.

https://github.com/Seneca-CDOT/telescope/blob/7368bcd49ed6cca1e36a3278c317a8c95f0d251e/src/api/status/public/index.html#L332

Reply:
This seems not done. @nguyenhung15913 can you please do it and then resolve this conversation?

Reply:
This need to be put inside a row, together with the feed card so they can look like the first 4 cards. Add the margin-top to the row container, not the cards.